lib/inferno/db/migrations/001_create_initial_structure.rb

```diff
@@ -161,5 +161,10 @@
       column :value, String
       index [:test_session_id, :name]
     end
+
+    create_table :requests_results do
```

Comment:
Sorry I didn't notice this before, but I think we're past the point of updating the original migration now, so this should go in a new migration. Once that is done and the swagger is updated I think this will be good to go.